docs/snap-clone.md

```diff
@@ -159,41 +159,89 @@ kubectl create -f snapshot.yaml
 
 ```console
 $ kubectl get volumesnapshot
-NAME               AGE
-rbd-pvc-snapshot   6s
+NAME               READYTOUSE   SOURCEPVC   SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS                            SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+rbd-pvc-snapshot   true         rbd-pvc                             1Gi           ocs-storagecluster-rbdplugin-snapclass   snapcontent-905e6015-2403-4302-8a4e-cd3bdf63507b   78s            79s
```

Comment:
`ocs-storagecluster-rbdplugin-snapclass` this is not the name in yaml files we have